### DISCUSSION TITLE
lens_model_DESIJ0136-0008 `EPL`

### DISCUSSION THREAD

**Discussion by Rajit13** [AstroBridge/BDLensing#223] [Ideas]
So I am still working on this model, I solved a few things with the suggestions I received earlier. Now I tried to switch the `EPL` lens with `SIE` and found this interesting occurrence below. Initially, I took the center-right objects as `EPL` galaxy. What should I do now, should I add both objects as `EPL` or continue with one being the `SIE`? Please also suggest to me which object I should choose as `EPL`.

![akbar](https://github.com/ajshajib/BDLensing/assets/81336262/0fa922fe-b68d-4343-94ed-cc831bc4cbaa)

**Comment by Rajit13** [comment]
@ajshajib @rafee-adnan @SulymanHossain any suggestions?

**Reply by rafee-adnan** [reply]
Can you share the converged value for both of the models?

**Reply by Rajit13** [reply]
For `EPL` center right 
![Untitled](https://github.com/ajshajib/BDLensing/assets/81336262/e1b371be-6251-4276-adc4-d8caa8335c26)

{'kwargs_lens': [{'theta_E': 1.814861252757777, 'gamma': 1.918386769902471, 'e1': -0.10864344124938255, 'e2': 0.07050640859687764, 'center_x': -0.6513394722072086, 'center_y': 0.13699026267373254}, {'gamma1': 0.022385196026953408, 'gamma2': 0.1500211196198303, 'ra_0': 0.0, 'dec_0': 0.0}, {'theta_E': 1.399818140879095, 'e1': -0.1101861728637632, 'e2': 0.02329004748302354, 'center_x': 0.9565261882828364, 'center_y': 0.5555852328891893}], 'kwargs_source': [{'amp': 13.064041596063547, 'R_sersic': 0.29786519789177063, 'n_sersic': 1.0, 'e1': 0.11089405221311192, 'e2': 0.00665675990922523, 'center_x': 0.34134308387823964, 'center_y': 0.41460154659975035}, {'amp': array([ 169.14506396,   59.60301268,   98.49645511,   80.96003208,
        -10.17975547,  114.54638793,  -51.70849885,    8.36345989,
       -191.3608843 , -158.3136176 ,  -92.60827877, -131.43285593,
         21.07521815, -119.52987369,   61.94835757,   -2.89965784,
         14.56586898,   11.28365402,  -16.50921039,    3.60433029,
        -18.89448222,  -34.77233997,  198.78919214,  -40.33859353,
        -50.88067886,    7.13774748,  -56.47788814,   -2.06154487,
       -312.70186713,  355.27978972, -224.71607369,  253.69739509,
        -18.81475959,   57.35160831,  118.33092004,   53.36875466,
       -238.52068039,   -0.53409125,  -12.10484463,   61.55495541,
         32.46975816,   56.01418962,   -5.46400609,  -50.1575604 ,
         -5.41087451, -168.48188278,    7.51028588,   15.07863   ,
         29.57237785, -214.96849004,  147.55954193,  -43.55774925,
        -55.91429425,   47.55718067,   -6.32479229, -112.80286962,
        230.04480235, -308.78213823,   93.63981252,   -9.85139228,
       -168.42050508, -135.20944107,  159.77952611,  -22.9339479 ,
         19.9188512 ,   90.90052876]), 'n_max': 10, 'beta': 0.030798947897522654, 'center_x': 0.34134308387823964, 'center_y': 0.41460154659975035}], 'kwargs_lens_light': [{'amp': 288.44130689708174, 'R_sersic': 1.0565589878907962, 'n_sersic': 4.0, 'e1': 0.03621520612273567, 'e2': 0.028992670933350187, 'center_x': -0.6513394722072086, 'center_y': 0.13699026267373254}, {'amp': -1700.2400496206155, 'R_sersic': 0.11824375968138112, 'n_sersic': 4.0, 'e1': 0.03621520612273567, 'e2': 0.028992670933350187, 'center_x': -0.6513394722072086, 'center_y': 0.13699026267373254}, {'amp': 308.9402691216539, 'R_sersic': 0.6295607592752112, 'n_sersic': 4.0, 'e1': -0.1101861728637632, 'e2': 0.02329004748302354, 'center_x': 0.9565261882828364, 'center_y': 0.5555852328891893}], 'kwargs_ps': [], 'kwargs_special': {}, 'kwargs_extinction': []}

**Reply by rafee-adnan** [reply]
And EPL for the left one?

**Comment by Rajit13** [comment]
Even after trying two EPL profiles, I am not getting suitable results. I need help with this model if anyone is free @aaimtiaz @SulymanHossain @rafee-adnan Here is the current code https://colab.research.google.com/drive/1cL_ihlxd3VC_RlO6cxoIwo6GI_r5Ut9u?usp=sharing

**Reply by SulymanHossain** [reply]
Although I'm currently busy with Zannat's model, I will try this model tomorrow. Also mentioning @Conjecture13, you can work with this model as your own model is done.

**Reply by mh-tanver** [reply]
Sure. I will try this.
